lib/jekyll-import/importers/rss.rb

```diff
@@ -51,7 +51,7 @@ def self.process(options)
             "title"  => item.title,
           }
 
-          header["tag"] = options["tag"] unless options.to_s.empty?
+          header["tag"] = options["tag"] unless (options["tag"].nil? || options["tag"].empty?)
```

Comment:
```suggestion
          header["tag"] = options["tag"] unless options.fetch("tag", {}).empty?
```

Reply:
@mattr- I didn't use `#fetch` because it only checks if `options.key?("tag")`. It will lead to a `nil` assignment when `options["tag"] = nil` is set explicitly.
The chances of the explicit *reset* are slim, albeit very much possible. I didn't want to divert way too much from the existing behavior.

Either ways, I'll go with your call.

Reply:
ah, good to know. let's keep it the way it is then.